modules/ROOT/pages/create-map-json-xml-to-outbound-x12 copy.adoc

```diff
@@ -0,0 +1,68 @@
+= Create a Map For XML or JSON to Outbound EDI X12
```

Comment:
```suggestion
= Create a Map for XML or JSON to Outbound EDI X12
```
Hmm...I wonder if we can or should replace "Create a Map for..." to just "Map"? 
Map XML or JSON to Outbound EDI X12?
If so, apply to similar topic titles in set.